### PR TITLE
Add code that makes forms un-live

### PR DESCRIPTION
**Pull request by chao-xian**
### What problem does this pull request solve?

Trello card: https://trello.com/c/E6iLOdFK/1180-add-a-un-alive-switch-for-a-form

If a form hits any sort of trouble, we need a reliable way to make it offline

### Things to consider when reviewing

- Ensure that you consider the wider context.
- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

### CODE REVIEW

File: app/models/form.rb
Comment (lfdebrux) [inline app/models/form.rb:1]: Is the `has_live_version` attribute used anywhere in forms-admin?

Comment (chao-xian) [reply app/models/form.rb:1]: [ARGH](https://github.com/search?q=repo%3Aalphagov%2Fforms-admin%20has_live_version&type=code)

Comment (chao-xian) [reply app/models/form.rb:1]: I'll drop that commit and do it later...

**Review by lfdebrux — APPROVED** [review]
Tested locally, works for me 👍

**Pull request merged**